components/layout/Navbar.tsx

```diff
@@ -28,7 +28,7 @@ const Navbar = () => {
 
   return (
     <nav className="absolute top-0 z-10 w-[100vw]">
```

Comment:
Is it possible to use `relative` positioning? Then we might not need to define `h-[105px]` on line 31. If relative positioning messes too many things up, then its fine to leave it.